sakai-evaluation-tool/src/webapp/component-templates/render_scaled_item.html

```diff
@@ -50,10 +50,13 @@ <h5 class="title">
 					</div>
 	            	-->
 				</h5>
+
+				<div style="clear: both"></div>
+
```

Comment:
I think you can just apply clearfix to the following div instead of this empty div